scripts/workers/submission_worker.py

```diff
@@ -337,9 +343,8 @@ def run_submission(challenge_id, challenge_phase, submission, user_annotation_fi
 
             leaderboard_data_list = []
             for split_result in submission_output['result']:
-
                 # get split_code_name that is the key of the result
-                split_code_name = split_result.items()[0][0]
+                split_code_name = list(split_result.keys())[0]
```

Comment:
I am not able to understand why is this change being done @RishabhJain2018 
As per my understanding, both will give the same result. Any warnings being thrown by Python3

Reply:
I have used the PR #1681 and built upon it.